assets/pages/paper-subscription-landing/paperSubscriptionLandingPage.jsx

```diff
@@ -49,6 +50,15 @@ const store = pageInit(reducer(method, promoInUrl), true);
 
 // ----- Render ----- //
 
+function flashSaleAwareHeading(): string {
```

Comment:
I think calling this `getHeading()` is more consistent with other parts of the code

Reply:
Thanks. I will rename this.